internal/tools/academiccalendar/model.go

```
@@ -0,0 +1,148 @@
+	if in.Authoritative && in.Source != SourceWUDA {
+		return ErrInvalid
+	}
+	for _, e := range in.Events {
+		if !validText(e.ID, MaxEventID) || !validText(e.Title, MaxEventTitle) || (e.Type != "" && !validText(e.Type, MaxEventType)) || (e.Description != "" && !validText(e.Description, MaxEventDescription)) || e.StartAt.IsZero() || !e.EndAt.After(e.StartAt) || !e.StartAt.Before(in.ValidUntil) || e.EndAt.After(in.ValidUntil) {
+			return ErrInvalid
+		}
+	}
+	return nil
+}
+
+func normalizeText(value string, max int) string {
```

golangci-lint (2.12.2)

[error] func normalizeText is unused

(unused)